# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/virtualbox/hotfix-vboxvideo.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- VirtualBox-7.2.14/src/VBox/Additions/x11/vboxvideo/Makefile.kmk.vanilla	2026-07-25 00:11:27.453970462 +0300
+++ VirtualBox-7.2.14/src/VBox/Additions/x11/vboxvideo/Makefile.kmk	2026-07-25 00:11:38.091089131 +0300
@@ -406,7 +406,7 @@
 
 ifdef VBOX_USE_SYSTEM_XORG_HEADERS
  # Build using local X.Org headers.  We assume X.Org Server 1.7 or later.
- DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS)) vboxvideo_drv_system
+ DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS))
  SYSMODS := $(filter-out vboxvideo_drv%,$(SYSMODS))
  vboxvideo_drv_system_TEMPLATE = VBoxGuestR3XOrgMod
  vboxvideo_drv_system_CFLAGS := \
